routes/project/location/postgres.rb

```diff
@@ -234,6 +234,28 @@ class Clover
         end
       end
 
+      r.post "set-maintenance-window" do
+        authorize("Postgres:edit", pg.id)
+
+        allowed_optional_parameters = ["maintenance_window_start_at"]
+        request_body_params = validate_request_params([], allowed_optional_parameters)
+
+        maintenance_window_start_at = request_body_params["maintenance_window_start_at"]
+        maintenance_window_start_at = nil if web? && maintenance_window_start_at == ""
```

Comment:
Why only for the web?

Reply:
` maintenance_window_start_at = nil` isn't needed if you move the validation code to the model (as mentioned in the earlier comment).  Sequel's typecasting will automatically convert `""` to `nil` since the column type is integer.